More2Life/MainButton.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  MainButton.swift
+//  More2Life
+//
+//  Created by Brendan Kingsford on 8/29/17.
+//  Copyright © 2017 More2Life. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class MainButton: UIButton {
```

Comment:
If you want this in a storyboard with @IBDesignable then you will want to put this in the Shared framework.